gameserver/index.ts

```diff
@@ -3,14 +3,29 @@ import {Deck} from "./GameBoard/Cards/Deck";
 import {CARDS} from "./GameBoard/Cards/Cards";
 import {CARD_VALUE} from "./GameBoard/Cards/CardValue";
 import {GameBoard} from "./GameBoard/GameBoard";
+import {getScore} from "./GameBoard/Cards/CardScore";
 
-let game = new GameBoard();
+function print(str: any) {
+    console.log(str);
+}
```

Comment:
unused function

Reply:
@lucalaissue the code in index was for testing the implementations, it will be changed with the socket.io implementation